WordPress/Classes/ViewRelated/NUX/LoginSocialErrorViewController.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import Gridicons
+
+class LoginSocialErrorViewController: UITableViewController {
+    fileprivate var errorTitle: String
+    fileprivate var errorDescription: String
+
+    fileprivate enum Sections: Int {
+        case titleAndDescription = 0
+        case buttons = 1
+
+        static var count: Int {
+            return buttons.rawValue + 1
+        }
+    }
+
+    fileprivate enum Buttons: Int {
+        case tryEmail = 0
+        case tryAddress = 1
+        case signup = 2
+    }
+
+    init(title: String, description: String) {
+        errorTitle = title
+        errorDescription = description
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        errorTitle = aDecoder.value(forKey: "errorTitle") as? String ?? ""
+        errorDescription = aDecoder.value(forKey: "errorDescription") as? String ?? ""
+
+        super.init(coder: aDecoder)
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        tableView.estimatedRowHeight = 100.0
+
+        view.backgroundColor = WPStyleGuide.greyLighten30()
```

Comment:
Seems like these properties would be better set on `viewDidLoad` since its work that should only need to be done once.